### DISCUSSION TITLE
关于通讯作者的显示，关于文献中科院分区显示的优化问题与建议。

### DISCUSSION THREAD

**Discussion by arcishin** [MuiseDestiny/zotero-style#679] [General]
![image](https://github.com/MuiseDestiny/zotero-style/assets/125451942/505e3590-7c1d-49cc-b4c3-b3ec296c732e)

很喜欢您的插件，但是介于一行的屏幕有限，想咨询是否支持将中科院1区这个“SCI基础版”几个字个字去除，还有SCIQ1中的“SCI”几个字去除。

另外鉴于通讯作者对于论文的查看非常重要，想咨询是否支持显示通讯作者在Creator中。或者另加一栏。

**Comment by cycle-space** [comment]
两个要求在**列设置**中修改都可以做到的
第一个问题：在“map”属性加上“SCI基础版=,SCI=,”
第二个问题：Format属性改为“${firstName}${lastName}”，Slices属性改为“0:1,-1”
